java-persistence/src/main/java/com/whitecloak/training/inventory/controller/ItemController.java

```diff
@@ -24,5 +28,37 @@ public ItemResource createItem(@RequestBody ItemForm form) {
         return itemService.create(form);
     }
 
+
+
     @GetMapping("/api/v1/items/{id}")
+    public ItemResource getItems(@PathVariable("id") Long id) {
+
+
+
+        try {
+            return itemService.getUser(id);
+        }catch (NullPointerException ex){
```

Comment:
In general java practise, you should not catch `NullPointerException` (or any subclass of `RuntimeException`).

These errors are easily captured in an `if` statement